server/api/store.go

```diff
@@ -117,8 +118,10 @@ type StoreInfo struct {
 }
 
 const (
-	disconnectedName = "Disconnected"
-	downStateName    = "Down"
+	alive        = "Alive"
+	disconnected = "Disconnected"
+	down         = "Down"
+	dead         = "Dead"
```

Comment:
I'm not sure if it is a good name. It may lead to some misunderstandings, as the tikv process may still be running.

Reply:
When it enters the dead status, the TiKV has already been Tombstone.

Reply:
May need some opinions from other reviewers.

Reply:
I think the down and disconnect is fuzzy for user if they do not see the code, and they may not known down will step to dead after the store has no regions.

Reply:
`Dead` is kind of arbitrary. Maybe a word like `Removed`, `Outcasted`, or `Detached` is a better choice to indicate that the TiKV node does not belong to the cluster anymore.